docs/source/caching/http-cache.mdx

```diff
@@ -2,81 +2,49 @@
 title: HTTP cache
 ---
 
-import {MultiCodeBlock} from 'gatsby-theme-apollo-docs';
 
-Apollo Android provides two different kinds of caches: an HTTP cache and a normalized cache. The HTTP cache is easier to set up but also has more limitations. This page focuses on the HTTP cache. If you want to deduplicate the storage of your objects and/or notify your UI when your data changes, take a look at the [normalized cache](./normalized-cache) instead.
+> This page focuses on the HTTP cache. If you want to deduplicate the storage of your objects and/or notify your UI when your data changes, take a look at the [normalized cache](./normalized-cache) instead.
 
-## HTTP cache
+> ℹ️ The HTTP cache is only available on the JVM.
 
-To enable HTTP cache support, add the dependency to your project's `build.gradle file. The latest version is
-[![Maven Central](https://img.shields.io/maven-central/v/com.apollographql.apollo3/apollo-api)](https://repo1.maven.org/maven2/com/apollographql/apollo3/)
+To enable HTTP cache support, add the dependency to your project's `build.gradle` file:
 
-```kotlin:title=build.gradle
+```kotlin:title=build.gradle[.kts]
 dependencies {
   implementation("com.apollographql.apollo3:apollo-http-cache:x.y.z")
 }
 ```
 
-### Raw HTTP response cache
+Then configure your HTTP cache:
 
-<MultiCodeBlock>
-
-```kotlin:title=Kotlin
-// Directory where cached responses will be stored
-val directory = File(cacheDir, "apolloCache")
-
-// Size in bytes of the cache
-val maxSize: Long = 1024 * 1024
-
-// Build the ApolloClient
-val apolloClient = ApolloClient.Builder()
-    .serverUrl("/")
-    .httpCache(directory = directory, maxSize = maxSize)
-    .build()
-                   
-// Control the fetch policy  
-val query = ApolloRequest.Builder(FeedQuery(limit = 10, type = FeedType.HOT))
-    .httpFetchPolicy(HttpFetchPolicy.CacheFirst)
-    .build()
-val dataResponse = apolloClient.query(query)
 ```
-
-```java:title=Java
-// Directory where cached responses will be stored
-File directory = new File(cacheDir, "apolloCache");
-
-// Size in bytes of the cache
-long maxSize = 1024 * 1024;
-
-// Build the ApolloClient
-ApolloClient apolloClient = HttpCacheExtensionsKt.httpCache(
-    new ApolloClient.Builder().serverUrl("/"),
-    directory,
-    maxSize
-  )
-  .build();
-Rx2ApolloClient rx2ApolloClient = new Rx2ApolloClient(apolloClient, Schedulers.io());
-
-// Control the fetch policy
-ApolloRequest<FeedQuery.Data> query = HttpCacheExtensionsKt.httpFetchPolicy(
-    new ApolloRequest.Builder<>(new FeedQuery(10, FeedType.HOT)),
-    HttpFetchPolicy.CacheFirst
-  )
-  .build();
-
-ApolloResponse<FeedQuery.Data> response = rx2ApolloClient.query(query).blockingGet();
+val dir = File(pathToCacheDirectory)
+
+// Configure a max size of 100MB
+val maxSize = 100 * 1024 * 1024
+
+apolloClient = ApolloClient.Builder()
+    .networkTransport(
```

Comment:
Shouldn't we show the `httpCache` extension?

Reply:
💯 , I forgot it existed